File: final-evaluation-generated-code/test-2/chatgpt.js
```javascript
const fetch = require('node-fetch');

let invokeAction = async () => {
    let response = await fetch('http://localhost:3000/http-express-calculator-simple/actions/subtract', {
        method: 'POST',
        headers: {
            'Content-Type': 'application/json'
        },
        body: JSON.stringify({
            input: 5
        })
    });

    if (response.ok) {
        let result = await response.json();
        console.log(result);
    } else {
        console.error(`Error: ${response.status}`);
    }
}

invokeAction();
```

File: final-evaluation-generated-code/test-7/chatgpt.py
import requests
import json

def read_property():
    url = "http://localhost:3000/http-express-calculator-simple/properties/result"
    headers = {
        'Content-Type': 'application/json',
    }
    response = requests.request("GET", url, headers=headers)
    return json.loads(response.text)

print(read_property())